lib/ansible/parsing/utils/addresses.py

```diff
@@ -126,7 +126,7 @@
 
     'ipv6': re.compile(
         r"""^
-            (?:{0}:){{7}}{0}|           # uncompressed: 1:2:3:4:5:6:7:8
+            ((?:{0}:){{7}}{0}|          # uncompressed: 1:2:3:4:5:6:7:8
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.